production/docker/config/loki-docker-memberlist-boltdb-shipper.yaml

```diff
@@ -0,0 +1,108 @@
+auth_enabled: false
+
+http_prefix:
+
+server:
+  http_listen_address: 0.0.0.0
+  grpc_listen_address: 0.0.0.0
+  http_listen_port: 3100
+  grpc_listen_port: 9095
+  log_level: debug
+
+memberlist:
+  join_members: ["loki-1", "loki_2", "loki_3"]
+  dead_node_reclaim_time: 30s
+  gossip_to_dead_nodes_time: 15s
+  left_ingesters_timeout: 30s
+  bind_addr: ['0.0.0.0']
+  bind_port: 7946
+
+ingester:
+  lifecycler:
+    join_after: 60s
+    observe_period: 5s
+    ring:
+      replication_factor: 2
+      kvstore:
+        store: memberlist
+    final_sleep: 0s
+  chunk_idle_period: 30m
+  max_chunk_age: 2h
+  chunk_retain_period: 10m
```

Comment:
This is a little too long and will lead to some higher memory usage, this config setting is not well documented unfortunately, but it's really important if you are using an index read cache.

If you are not using an index (which this example is not), then we should keep the value small, `chunk_retain_period: 30s`

Reply:
Fixed